0.32.0 `test_vclock_default` and `test_vclock_extra_lines` fail

### Issue description

Hi! 👋 

When upgrading to 0.32.0 on Arch Linux I ran into `test_vclock_default` and `test_vclock_extra_lines` failing:

```
=================================== FAILURES ===================================
__________________________ test_vclock_default[1-x11] __________________________

vclock_manager = <test.helpers.TestManager object at 0x7f0f2d630c30>

    def test_vclock_default(vclock_manager):
>       assert vclock_manager.c.widget["verticalclock"].info()["text"] == "10|20"
E       AssertionError: assert 'N/A|N/A' == '10|20'
E
E         - 10|20
E         + N/A|N/A

test/widgets/test_vertical_clock.py:98: AssertionError
---------------------------- Captured stdout setup -----------------------------
2025-06-14 16:19:55,532 libqtile sleep.py:_start():L104  Could not find logind service. Suspend and resume hooks will be unavailable.
2025-06-14 16:19:55,532 libqtile manager.py:reconfigure_screens():L450  Reconfiguring screens.
________________ test_vclock_extra_lines[1-x11-vclock_manager0] ________________

vclock_manager = <test.helpers.TestManager object at 0x7f0f2d6305f0>

    @config(format=["%H", "%M", "-", "%d", "%m", "%Y"])
    def test_vclock_extra_lines(vclock_manager):
>       assert vclock_manager.c.widget["verticalclock"].info()["text"] == "10|20|-|01|01|2024"
E       AssertionError: assert 'N/A|N/A|N/A|N/A|N/A|N/A' == '10|20|-|01|01|2024'
E
E         - 10|20|-|01|01|2024
E         + N/A|N/A|N/A|N/A|N/A|N/A

test/widgets/test_vertical_clock.py:103: AssertionError
---------------------------- Captured stdout setup -----------------------------
2025-06-14 16:19:55,764 libqtile sleep.py:_start():L104  Could not find logind service. Suspend and resume hooks will be unavailable.
2025-06-14 16:19:55,764 libqtile manager.py:reconfigure_screens():L450  Reconfiguring screens.
=========================== short test summary info ============================
FAILED test/widgets/test_vertical_clock.py::test_vclock_default[1-x11] - AssertionError: assert 'N/A|N/A' == '10|20'

  - 10|20
  + N/A|N/A
FAILED test/widgets/test_vertical_clock.py::test_vclock_extra_lines[1-x11-vclock_manager0] - AssertionError: assert 'N/A|N/A|N/A|N/A|N/A|N/A' == '10|20|-|01|01|2024'

  - 10|20|-|01|01|2024
  + N/A|N/A|N/A|N/A|N/A|N/A
====== 2 failed, 2081 passed, 84 skipped, 4 xpassed in 1000.58s (0:16:40) ======
```

I am not sure what goes wrong here exactly.
However, here are also the full build and check logs:

[qtile-0.32.0-1-x86_64-build.log](https://github.com/user-attachments/files/20741589/qtile-0.32.0-1-x86_64-build.log)
[qtile-0.32.0-1-x86_64-check.log](https://github.com/user-attachments/files/20741587/qtile-0.32.0-1-x86_64-check.log)


### Version

0.32.0

### Backend

X11 (default)

### Config

```python
n/a
```

### Logs

```text

```

### Required

- [x] I have searched past issues to see if this bug has already been reported, and it hasn't been.
- [x] I understand that people give their precious time for free, and thus I've done my very best to make this problem as easy as possible to investigate.

--- a/test/widgets/test_vertical_clock.py
+++ b/test/widgets/test_vertical_clock.py
@@ -24,8 +24,8 @@ from importlib import reload
 import pytest
 
 from libqtile.config import Bar, Screen
-from libqtile.confreader import Config
 from libqtile.widget import vertical_clock
+from test.helpers import BareConfig
 
 
 # Mock Datetime object that returns a set datetime and also
@@ -71,7 +71,7 @@ def patched_clock(monkeypatch):
 
 @pytest.fixture(scope="function")
 def vclock_manager(manager_nospawn, request, patched_clock):
-    class VClockConfig(Config):
+    class VClockConfig(BareConfig):
         screens = [
             Screen(
                 left=Bar(

